README.md

```diff
@@ -209,18 +204,17 @@ App\Entity\User:
         city: <dictionary('cities')>
 ```
 
-## Create your own dictionary implementation
+## Create your own dictionary implementation
 
 ### Dictionary
 Your dictionary implementation must implements the interface [Dictionary](src/Knp/DictionaryBundle/Dictionary/Dictionary.php).
 
-### Dictionary Factory
+### Dictionary Factory
 You must create a dictionary factory that will be responsible to instanciate your dictionary.
 
 ```yaml
 services:
-    app.dictionary.factory.my_custom_factory:
```

Comment:
Maybe keep this example for symfony 2.8 support as mentionned in requirements ?

Reply:
Fixed 👍